architecture/README.md

```diff
@@ -1 +1,147 @@
-# Architecture
\ No newline at end of file
+# Architecture
+
+The Open Sovereign AI Cloud solution gives cloud providers a complete platform
+to offer self-service provisioning of compute infrastructure, including VMs and
+OpenShift clusters, integrated with the cloud provider’s existing infrastructure
+components. It features a fulfillment workflow that is powerful enough to handle
+complex cluster deployment while also being flexible enough to accommodate
+future expansion into other types of deployments.
+
+## Templates
+
+Self-service provisioning in OSAC is built around a concept of Templates.
+Whether an end user wants to provision a VM, a cluster, or something else,
```

Comment:
Just a note that I don't think we're planning on using templates for bare metal fulfillment; I don't know if that's important enough to call out as a special case?

Reply:
I don't really think so; really depends what the definition of a template is; if its an ansible role... and there where alternatives, why wouldn't it apply?

Reply:
Because there's other implications to being a template beyond just being an ansible role; there's a degree of customizability where a service provider can create their own templates associated with their own custom ansible roles. As far as I'm aware, we're not planning to have that level of flexibility for bare metal fulfillment.

Reply:
Let's discuss this more. If we're not going to use templates for bare metal, we should probably write down why.

Reply:
I think this has now changed, as per conversation friday; i.e., we are getting rid of bare metal fullfillment, and having VDC fullfillment; were we create a virtual data center that can include virtual or physical machines connected on networks.  In that case, there are certainly templates involved; actually hierarchical ones.